packages/runtime/src/dataViews/DataViewExpander.ts

```diff
@@ -562,6 +567,32 @@ export class DataViewExpander {
                     response: responseItemDVO
                 } as CreateAttributeRequestItemDVO;
 
+            case "DeleteAttributeRequestItem":
+                const deleteAttributeRequestItem = requestItem as DeleteAttributeRequestItemJSON;
+                const localAttributeResultForDelete = await this.consumption.attributes.getAttribute({ id: deleteAttributeRequestItem.attributeId });
+                const localAttributeDVOForDelete = await this.expandLocalAttributeDTO(localAttributeResultForDelete.value);
+
+                if (isDecidable) {
+                    return {
+                        ...deleteAttributeRequestItem,
+                        type: "DecidableDeleteAttributeRequestItemDVO",
+                        id: "",
+                        name: requestItem.title ? requestItem.title : "i18n://dvo.requestItem.DecidableConsentRequestItem.name",
+                        isDecidable,
+                        response: responseItemDVO,
+                        attribute: localAttributeDVOForDelete
+                    } as DecidableDeleteAttributeRequestItemDVO;
+                }
+                return {
+                    ...deleteAttributeRequestItem,
+                    type: "DeleteAttributeRequestItemDVO",
+                    id: "",
+                    name: requestItem.title ? requestItem.title : "i18n://dvo.requestItem.ConsentRequestItem.name",
```

Comment:
`"i18n://dvo.requestItem.DeleteAttributeRequestItem.name"` instead of `"i18n://dvo.requestItem.ConsentRequestItem.name"`